src/installation.md

```diff
@@ -16,8 +16,19 @@ For an easy installation simply click on the **Windows** button under **BETA**.
 
 Once that's is all finished, you are ready to [get started with the DM language!](./hello_world.md)
 
-## VSCode (Optional)
+## VSCode (Optional but Highly Recommended)
```

Comment:
```suggestion
## VSCode (Optional but Very Very Highly Recommended)
```
let's excargate.

Reply:
just omit (optional) entirely, it's realistically not, especially on goon's pathing.

Lying to people is important in noob guides.